src/cli-client/snap/snapcraft.yaml

```diff
@@ -0,0 +1,31 @@
+name: oci-factory # TODO 'snapcraft register <name>'
+base: core22
+version: '0.0.1'
+summary: The OCI Factory CLI client to build, upload and release OCI images
+description: |
+  The OCI Factory CLI client is a tool that builds, tests, and releases
+  the OCI images owned by Canonical using the Github workflow in 
+  the OCI Factory repository.
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: strict
+
+parts:
+  oci-factory:
+    plugin: go
+    build-snaps: [go/latest/stable]
+    source: https://github.com/canonical/oci-factory
+    source-type: git
+    source-subdir: src/cli-client/cmd/oci-factory
+    stage-packages:
+      - git
```

Comment:
why is this needed?

Reply:
We need to call the `git rev-parse --show-prefix ` command as a subprocess in the CLI client as the git library does not support `git rev-parse` yet.

Reply:
got it. That means `git` should also be a dependency for those who are installing from source (thus part of the inner README)

Reply:
Yes. I will add it.

Reply:
Other suggested changes on documentation and this is fixed in [49ac137](https://github.com/canonical/oci-factory/pull/198/commits/49ac137eb8fcb59588bf62394ed677cc9f43f7a4).